test/integration/user-management-api/services/proxy/policies_test.rb

```diff
@@ -58,6 +58,11 @@ def test_invalid_policies_config
     assert_response :unprocessable_entity
   end
 
+  def test_invalid_json_policies_config
+    put admin_api_service_proxy_policies_path(valid_params.merge({ proxy: { policies_config: { name: 'echo '}.to_json }}))
+    assert_response :unprocessable_entity
+  end
```

Comment:
This should also test that the response.body contains the proper error message in policies_config

Reply:
done.